### ISSUE TITLE
3.3.0 stopped scanning correctly

### ISSUE THREAD

**Issue by suloku** [farkam135/GoIV#563]
3.2.0 correctly detected species, new version doesn't. I'm not sure how I can help debuggin (screen captureof my device?).

PoGo: 0.4.5
Nexus 4 running CyanogenMod 13, configured to 320 DPI (default).

UPDATE: this is when using manual screen capture, screen recording mode works better, but has number recognition problems mentioned in other issues.

UPDATE 2: level 30 pokemon are recognized as level 30.5 too.

Also, I love the perfect IV CP% feature, would it be possible (if it fits) to print the actual CP value or the difference? I.e. a 20% loss meaning 50 CP isn't the same as a 15% loss that means 75 CP loss.

**Comment by NightMadness** [comment]
Hi @suloku thank you for reporting
If you can please make sure no screen color chagers are on and post a screenshot from your device.

**Comment by EdwarDDay** [comment]
I have the same problem with my Nexus 4 - Android 5.1.1
This is the screenshot of my Vaporeon:
![screenshot_2016-11-23-13-49-57](https://cloud.githubusercontent.com/assets/4127904/20562751/87b388cc-b185-11e6-9e87-4127dc55a144.png)
And that is the result from GoIV (3.3.1)
![screenshot_2016-11-23-10-23-10](https://cloud.githubusercontent.com/assets/4127904/20562750/8788623c-b185-11e6-846c-608d33ea687c.png)

**Comment by NightMadness** [comment]
Hi @EdwarDDay thank you for reporting the issue. 

Did scanning work in previous versions?

I see there are two black borders covering top and bottom of the screenshot, is your PoGo not running throuhout the whole length of the screen?

**Comment by EdwarDDay** [comment]
Scanning worked in version 3.2.0 and the black borders just cover unnecessary information.
I heard from other games, that people steal accounts from too much information and don't want to give it a chance. I could have cut it, but than the aspect ratio would be wrong.

**Comment by nahojjjen** [comment]
If the pokemon is identified as wrong species & the values are blank, then there's an issue with the ocrHelper class. 

That level 30 pokemon are scanned as 30.5 is a separate issue that we're aware of.

Do you know how to use debug mode in android studio? In case you do, put a breakpoint in the hp scanning method in the OcrHelper class and look at the hp bitmap - check if it's cropped correctly.

**Comment by EdwarDDay** [comment]
I checked the HP bitmap and it was wrong.
Than I tested GoIV with screen scan and that works. I looked at the produced images and the screen scan method bitmaps don't have the navigation bar (the three buttons at the bottom of the screenshot), while the screenshots from the system still have the navigation bar and are scaled with navigation bar. So the image ratio is wrong.
Maybe the screenshot should be ratio consistent scaled.
